socorro/processor/rules/mozilla.py

```diff
@@ -87,6 +89,12 @@ def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
         processed_crash["ReleaseChannel"] = raw_crash.get("ReleaseChannel", "")
         processed_crash["build"] = raw_crash.get("BuildID", "")
 
+        # NOTE(willkg): ApplicationBuildID is for Fenix which sends the gecko view build
+        # id in BuildID.
```

Comment:
Fenix is the only product that does this, but it's possible other products may also do this going forward. I don't remember where the data that populates `ApplicationBuildID` comes from specifically. We can adjust the comment when we know more.